components/infra-proxy-service/server/proxy.go

```diff
@@ -30,6 +31,51 @@ type ChefClient struct {
 	client *chef.Client
 }
 
+// SearchObjectsWithDefaults constructs the search query with defaults params values
+// Default per_page is set 1000
+// Default search term q is "*:*"
+// Default page is set to 0
+func (c *ChefClient) SearchObjectsWithDefaults(searchIndex string, searchQuery *request.SearchQuery, params map[string]interface{}) (*chef.SearchResult, error) {
```

Comment:
Are we using the same logic we had introduced in the Server list/Roles List Pages? If so, we should refactor the code to avoid duplicity.

Reply:
Yes, this PR also contains refactor of other search list pages, This is only meant to avoid duplicity,